testing/regress/ecl/normalize4.ecl

```diff
@@ -67,9 +67,9 @@ streamed dataset(out1Rec) extractResult3(doneRec done) := BEGINC++
         }
         virtual void stop() {}
     private:
+        Linked<IEngineRowAllocator> allocator;
```

Comment:
Yes - without it you get a warning (treated as an error) about the initialization order specified in the constructor not matching the order that it will actually happen in.